### DISCUSSION TITLE
Permissions Request Notifications

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#3676] [Various ideas and suggestions]
I have one app that launches another app. I have to allow some communication between them if I want both of them or either of them sandboxed, but how do I know which communication to allow and which restrict to loosen? IPC? Window? It would be nice if Sandboxie could inform users that whichever app running within Sandboxie requests permission to communicate with whichever other app or needs whichever resource from whichever other app. A good example is when apps launch default browsers to perform a login or when opening a magnet link that asks if users wish to open magnet link with whichever external app. Without knowing which communication to allow between their sandboxes, it ends up being a painful game of trial and error.

**Comment by bastik-1001** [comment]
This could be a feature request. It has some overlap with #3144, but what you are looking for is broader and seems to be useful to assist users in configuring their sandboxes.
